src/platform/common/stm32/pwm_output_dshot_shared.c

```diff
@@ -40,15 +40,17 @@
 #include "stm32f4xx.h"
 #endif
 
-#include "pwm_output.h"
+#include "drivers/pwm_output.h"
 #include "drivers/dshot.h"
-#include "drivers/dshot_dpwm.h"
+#include "dshot_dpwm.h"
 #include "drivers/dshot_command.h"
 #include "drivers/motor.h"
 
 #include "pwm_output_dshot_shared.h"
 
 FAST_DATA_ZERO_INIT uint8_t dmaMotorTimerCount = 0;
+FAST_DATA_ZERO_INIT uint8_t motorCount = 0;
```

Comment:
`motorCount` is global here, but `static` in other platform files (and also local variable or parameter in some functions). It is extremely confusing.

Reply:
In practice we really only need the one motorCount... I will fix this up.